.travis.yml

```
@@ -0,0 +1,35 @@
+language: php
+
+env:
+  global:
+  - COMPOSER_ROOT_VERSION=2.x-dev
+
+matrix:
+  include:
+  - php: 5.6
+    env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
+  - php: 7.0
+    env: DB=MYSQL PHPUNIT_TEST=1
+  - php: 7.1
+    env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+  - php: 7.2
+    env: DB=MYSQL PHPUNIT_TEST=1
+
+before_script:
+# Init PHP
+- phpenv rehash
+- phpenv config-rm xdebug.ini
+- echo 'memory_limit = 2048M' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
+# Install composer dependencies
+- composer validate
+- composer require --no-update silverstripe/recipe-cms:1.x-dev cwp/starter-theme:2.x-dev silverstripe/fulltextsearch:^3
```

```suggestion
- composer require --no-update silverstripe/recipe-cms:1.x-dev cwp/starter-theme:2.x-dev
```

Don't think we need full text search to test this.